x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/data_tier_allocation_field/data_tier_allocation_field.tsx

```diff
@@ -73,15 +73,19 @@ export const DataTierAllocationField: FunctionComponent<Props> = ({ phase, descr
         /**
          * We'll drive Cloud users to add a cold tier to their deployment if there are no nodes with the cold node role.
```

Comment:
Could you please update the comment as well to include both cold and frozen tiers?